src/main/java/de/konfidas/ttc/validation/Validator.java

```diff
@@ -0,0 +1,21 @@
+package de.konfidas.ttc.validation;
+
+import de.konfidas.ttc.exceptions.ValidationException;
+import de.konfidas.ttc.tars.LogMessageArchive;
+
+import java.util.Collection;
+
+
+/**
+ *  A Validator validates aspects of TAR-Files. For example the sequence of Signature Counters,
+ *  the plausibility of time stamps w.r.t to the Sequence of Signature Counters, or the plausibility of
+ *  transaction numbers.
+ *
+ *  A Validator has a state, that it keeps, so calling validate() multiple times can create different results.

```

Comment:
Über diese States würde ich mal gerne mit dir telefonieren. Bin mir unsicher, ob ich das richtig verstehe

Reply:
Well, you know my number :) General Idea for the interested Audience:

TSE user uses TSE
TSE User exports Tar -> tar1.tar
TSE User deletesstoredData 
TSE user uses TSE
TSE User exports Tar -> tar2.tar

Now, tar1.tar and tar2.tar together form a sequence of tars, which contain a complete sequence of signature counter values, transaction counter values, etc. Probably some transactions where opened in tar1.tar and updated / closed in tar2.tar

This makes it feasible to be able to verify such sequence of tars somehow  "together". 
This is, what validators make possible.
```

Validator val = new SignatureCounterValidator();

val.validate(tar1);
val.validate(tar2);

```
does the trick. The Validator has internal state, which allows him to detect, where the signature counter was and what (at the end of tar1, when `val.validate(tar1)` is done) and continues there, when ` val.validate(tar2)` starts. 

In addition, the Validator keeps track of the state w.r.t. the TSE serial number. So oneValidator can validate tars of multiple TSE, without getting lost / confused here. 

Note that currently the validators will produce errors, when the tar files are not disjoint, i.e. in the example above, if the TSE user does not delete the stored tse data and validate both tars, he will recieve a lot of errors, because all log messages of tar1 are also part of tar2, which leads to a lot instances of `SignatureCounterDuplicateException`